### PR TITLE
Fix UFC coordinate input for preplanned targets on F/A-18

### PR DESCRIPTION
**Pull request by mzltn**
Changes:
 - Add methods to PrePlannedCoordinate to extract values from coordinate: Convert_to_DegMin, Convert_to_DegMinSec.
 - Refactor BuildCoordinate method (namespace DTC.Models.FA18.Upload, class PrePlannedBuilder).

### CODE REVIEW

**Comment by Gosling616** [comment]
> Changes:
> 
>  - Add methods to PrePlannedCoordinate to extract values from coordinate: Convert_to_DegMin, Convert_to_DegMinSec.
> 
>  - Refactor BuildCoordinate method (namespace DTC.Models.FA18.Upload, class PrePlannedBuilder).

**Comment by Gosling616** [comment]
The Preplanned coordinates are correct in that the numbers are the correct numbers to input into the JDAMs. These numbers are already in DD MM SS.ss however, they are displayed incorrectly as DD MM.mmmm.  But they are NOT Decimal Minute numbers. They are Decimal Second numbers already.

**Comment by mzltn** [comment]
Thanks for the review. I have somehow overlooked this case. I have checked, and the coordinate capture feature also gives the same format that you mentioned.
In this case however, I think the TextBox mask should be set to follow the same format, as the current one is somewhat misleading.

File: dcs-dtc/Models/FA18/PrePlanned/PrePlannedStation.cs [hunk 1]
```diff
@@ -10,18 +10,17 @@ public class PrePlannedStation
     {
         public StationType stationType {  get; set; }  
 
-        public int stationNumber { get; }
-        public PrePlannedCoordinate PP1 {  get; set; }  
-        public PrePlannedCoordinate PP2 {  get; set; }  
-        public PrePlannedCoordinate PP3 {  get; set; }  
-        public PrePlannedCoordinate PP4 {  get; set; }  
-        public PrePlannedCoordinate PP5 {  get; set; }  
+        public int stationNumber { get; set; }
```
Comment (p-louis) [inline dcs-dtc/Models/FA18/PrePlanned/PrePlannedStation.cs:13]: stationNumber should not have a setter as it should be fixed once initialized

**Review by p-louis — APPROVED** [review]
Hey! Thanks for cleaning up the mess I made with the PP stuff 😃 

And sorry I didn't react sooner, I usually don't check for new PRs on this repo because most people open theirs on the upstream one.

Overall LGTM

**Pull request merged**